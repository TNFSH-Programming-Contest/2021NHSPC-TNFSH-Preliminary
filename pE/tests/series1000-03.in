999 998
3 4 20
4 5 16
6 3 16
5 7 13
8 9 24
7 8 25
10 6 7
11 12 21
9 13 15
12 10 6
14 11 8
15 14 9
16 17 17
18 15 16
17 19 30
19 18 10
20 16 16
21 22 22
23 24 22
22 20 13
13 25 28
26 27 7
25 26 22
28 21 26
29 28 19
30 31 21
32 33 5
34 30 5
35 34 6
24 36 12
37 35 9
38 39 12
27 38 25
40 23 5
31 29 15
39 41 10
36 37 16
42 43 9
44 45 14
46 32 28
47 46 14
48 44 15
49 50 14
51 52 21
53 54 16
55 42 20
45 47 9
56 57 26
58 49 10
59 55 24
57 60 19
61 51 9
62 63 15
41 64 19
52 56 13
65 66 7
60 65 28
64 67 24
68 61 7
33 40 24
69 48 17
67 70 9
71 72 17
73 59 26
72 68 7
43 62 18
50 69 28
54 74 21
63 75 25
74 76 22
77 78 11
79 58 13
80 81 9
78 82 20
70 83 18
84 53 30
82 85 7
86 77 12
83 80 26
85 87 20
75 84 8
88 73 17
89 90 12
87 88 7
91 92 7
93 89 8
94 95 29
92 93 14
76 96 30
97 98 26
95 99 29
66 91 15
90 79 7
99 100 9
101 86 27
100 102 8
98 103 7
81 104 19
105 94 17
96 106 13
107 108 20
109 97 29
103 71 30
104 107 26
106 110 10
110 109 5
111 101 30
102 111 8
112 113 12
108 112 12
113 114 16
115 105 27
116 117 24
118 119 18
117 120 12
121 122 28
123 116 26
124 123 10
114 125 27
119 126 15
126 121 25
120 127 27
128 129 24
122 128 24
129 124 18
125 130 20
127 131 5
132 133 18
134 135 13
135 118 23
136 134 14
130 136 16
137 138 23
139 137 29
131 140 22
141 142 10
133 143 26
143 144 9
145 139 13
146 145 9
138 147 16
144 141 22
148 146 13
149 150 17
147 115 11
151 148 27
142 151 28
152 132 23
153 152 5
154 155 18
155 153 13
156 149 24
157 158 24
140 156 23
150 157 7
158 159 24
159 160 23
160 161 26
162 163 10
163 154 12
164 162 19
165 164 19
161 166 29
167 168 10
168 169 22
166 170 5
171 167 21
172 165 26
173 174 23
175 176 5
177 178 10
170 171 18
169 177 18
179 180 11
181 172 25
182 183 14
184 185 11
186 182 12
174 187 11
188 181 11
189 173 29
185 175 15
190 191 17
192 193 24
178 192 18
176 179 19
191 194 10
193 195 25
196 197 26
183 198 20
180 190 8
198 199 27
200 188 7
201 202 23
203 204 24
205 189 12
202 205 25
206 207 25
208 196 25
207 209 30
199 210 24
204 208 23
211 186 11
212 213 12
214 215 20
216 217 20
218 216 23
219 206 25
215 211 12
220 219 30
210 212 27
221 218 15
222 201 12
223 224 29
225 221 17
195 220 27
226 223 23
194 227 29
228 214 15
229 225 13
217 230 7
213 231 25
230 226 18
187 232 18
227 233 28
234 229 12
235 236 6
237 238 22
238 222 23
239 240 12
197 200 29
241 234 8
242 241 18
243 244 29
245 246 19
247 248 28
231 249 29
232 250 16
251 245 21
236 252 12
252 253 11
244 242 22
254 247 10
255 256 17
256 228 21
248 257 24
246 258 21
209 259 9
259 235 21
258 260 16
261 237 20
262 263 22
250 264 25
265 262 24
266 267 11
268 261 13
257 203 18
240 266 26
269 270 28
271 251 29
272 273 8
273 243 7
270 255 5
274 254 7
275 276 26
277 239 16
276 268 17
278 279 7
279 275 24
267 280 24
281 265 29
282 283 20
284 277 5
260 282 14
285 274 9
286 284 29
233 287 19
288 289 7
290 291 25
280 292 5
253 293 24
224 286 5
294 295 23
296 272 27
287 296 13
297 298 16
299 300 23
289 285 29
301 299 13
302 269 5
295 288 27
303 297 10
304 305 25
306 184 26
307 308 17
309 310 5
305 311 21
264 294 16
312 313 15
313 303 27
314 309 29
311 290 24
315 314 9
316 315 6
263 317 22
318 319 26
291 316 23
317 320 12
321 312 6
320 322 19
323 324 22
325 326 25
327 328 23
319 323 20
293 329 20
330 318 12
322 331 13
332 333 23
333 278 23
249 334 12
331 271 23
335 336 8
337 327 28
336 302 7
324 338 6
283 321 14
339 340 20
341 342 7
292 330 10
298 343 15
344 345 27
300 346 23
347 344 20
348 349 21
350 351 6
338 352 30
351 341 14
353 354 10
334 355 20
356 357 17
346 358 21
359 360 8
343 361 30
362 363 7
364 337 19
355 301 8
365 366 10
367 353 23
363 368 5
369 370 22
371 335 26
372 339 19
373 374 28
375 359 8
358 304 8
310 376 19
342 365 29
377 306 11
329 378 16
379 364 19
378 350 9
380 381 9
382 348 24
357 371 21
383 356 14
384 385 16
386 387 7
388 389 9
352 384 8
308 390 22
391 392 19
370 372 8
345 391 30
360 380 11
393 394 6
395 383 7
326 396 30
374 347 20
390 397 25
398 325 10
366 362 25
328 367 5
399 400 22
381 401 9
402 375 14
403 382 10
404 307 12
349 405 17
376 406 16
407 373 17
401 408 27
405 395 12
409 410 25
400 402 30
361 411 14
412 332 6
396 407 21
340 409 5
397 369 22
413 379 28
406 414 8
415 416 25
414 417 9
418 404 5
419 420 19
408 393 30
421 399 28
389 422 20
387 281 27
411 423 16
424 377 17
425 426 28
427 421 14
385 428 5
368 388 25
429 419 19
420 425 6
417 427 11
430 415 27
392 418 18
431 432 26
394 430 20
423 433 9
434 413 5
435 424 7
436 437 21
433 431 30
410 429 17
416 436 7
438 403 21
439 434 28
440 398 11
441 439 25
354 442 17
422 443 8
426 412 14
444 445 5
443 446 20
447 448 30
449 450 15
442 438 7
448 440 9
450 451 10
445 452 18
453 441 21
454 455 12
451 456 17
457 447 23
452 454 7
437 435 14
446 449 16
428 458 28
432 459 28
460 457 8
456 461 8
461 453 5
458 462 13
463 444 23
464 465 10
466 386 17
467 466 12
468 469 21
469 470 11
471 472 15
472 468 24
465 473 22
462 467 26
473 471 26
474 475 29
476 460 7
459 474 11
455 477 16
478 476 17
470 478 26
477 479 27
480 481 12
481 463 11
475 482 26
483 484 27
479 485 10
486 487 13
488 489 7
489 483 16
490 486 15
491 492 19
493 480 12
494 495 27
492 496 29
484 494 10
497 491 10
498 499 21
500 501 10
495 464 9
502 488 13
503 504 25
501 493 11
505 506 27
482 507 13
508 498 21
509 503 18
510 511 5
506 508 9
511 505 12
504 512 19
487 509 21
513 514 19
512 515 19
516 510 13
517 518 26
519 490 28
520 521 6
522 523 17
524 525 19
526 527 6
528 497 29
521 526 10
529 502 13
530 524 10
496 531 5
532 516 13
533 528 22
534 522 23
499 534 14
535 536 22
537 538 17
531 500 24
539 540 28
536 541 14
542 513 14
543 544 30
545 546 17
547 517 26
548 549 22
514 550 14
551 552 10
485 532 15
544 545 28
550 553 30
515 535 11
554 548 11
555 556 5
549 529 23
557 558 26
559 560 16
556 539 29
561 559 25
562 563 21
564 565 7
560 566 19
567 568 12
566 519 23
569 567 13
570 533 12
558 554 22
571 555 13
572 573 18
552 572 12
574 561 23
568 574 5
573 557 24
575 569 25
576 537 28
525 577 17
578 571 7
546 579 12
565 580 7
581 542 21
563 582 19
553 583 7
580 551 28
579 584 29
585 570 9
586 587 24
587 588 12
507 589 9
540 590 24
577 578 24
527 591 19
582 592 19
538 593 27
541 594 27
588 595 29
523 596 14
597 564 20
593 598 23
589 599 21
518 600 30
592 601 24
602 603 13
604 530 9
583 605 29
606 602 9
598 607 6
595 604 18
608 586 6
609 585 25
610 547 9
605 608 20
600 576 17
601 611 15
612 562 26
596 613 29
584 614 23
611 609 10
599 612 11
614 615 24
591 616 15
615 617 26
618 575 30
619 597 6
616 620 12
620 621 8
617 606 22
622 581 17
594 543 6
623 618 12
607 624 30
625 622 15
624 619 15
603 626 29
627 628 23
629 630 23
631 627 17
628 625 19
590 520 18
632 610 5
633 631 15
630 633 24
634 629 29
635 636 30
636 637 29
613 638 9
626 639 15
639 635 18
638 623 17
640 632 13
637 634 15
641 642 23
643 644 11
621 643 21
644 645 19
646 641 25
647 648 24
649 647 30
642 649 5
648 650 25
651 646 14
652 651 29
650 653 22
654 655 24
656 654 18
657 652 17
658 640 7
645 659 9
660 661 17
653 656 10
655 662 26
663 664 10
664 657 11
662 660 17
661 658 16
665 663 14
666 667 7
668 666 22
669 670 24
671 672 20
673 671 18
670 674 6
675 676 26
674 677 16
678 679 21
680 675 20
676 681 13
682 668 16
659 669 26
683 678 12
684 685 30
686 673 20
677 682 23
681 684 26
687 688 23
689 690 18
691 665 22
692 687 9
688 693 23
685 694 28
672 680 21
693 695 15
679 692 25
696 689 26
697 683 18
698 699 25
700 698 21
701 700 28
667 702 11
694 703 12
702 686 13
704 705 30
699 706 29
707 691 5
690 701 11
708 709 25
710 711 24
712 708 29
713 710 5
706 714 27
711 697 6
705 715 28
695 707 7
716 717 5
717 718 7
719 720 17
721 704 26
722 723 11
724 725 30
726 727 26
728 729 15
723 728 18
718 713 15
709 724 28
725 730 15
714 731 21
715 732 18
729 719 6
703 733 22
732 734 24
735 712 18
733 696 5
730 716 28
736 737 9
737 726 8
727 738 16
734 739 10
740 721 19
738 722 19
741 735 9
742 743 18
744 741 23
745 746 30
743 747 6
748 745 5
749 748 26
747 744 8
750 751 7
752 742 29
753 754 20
731 755 9
756 750 24
751 757 11
755 753 13
720 752 20
757 758 12
746 740 21
759 760 9
739 761 6
758 762 29
762 736 30
760 763 15
764 756 20
765 749 24
766 765 23
763 767 28
768 769 6
770 771 10
772 773 25
769 774 6
773 775 16
776 768 27
777 778 26
779 780 12
775 781 17
782 783 16
784 776 9
785 764 10
778 786 7
787 759 24
783 788 6
771 789 18
774 790 16
791 792 6
789 787 28
793 794 26
790 766 7
794 795 21
796 797 19
798 785 21
761 799 27
792 800 28
788 798 6
800 801 28
795 802 17
803 804 11
805 806 21
807 808 12
809 779 16
810 770 12
811 805 29
812 811 8
813 791 23
814 796 24
767 807 7
815 810 16
781 816 10
799 814 20
817 782 20
818 817 6
808 772 15
819 820 27
821 793 23
820 784 10
802 822 17
823 821 25
797 812 16
804 824 24
806 825 24
825 813 29
780 818 26
824 815 23
801 826 5
822 803 6
754 827 13
828 829 21
830 777 7
831 809 30
832 833 8
827 819 25
834 830 16
833 835 10
836 837 9
786 828 17
826 832 23
837 838 26
816 834 26
839 836 5
835 839 26
838 840 27
841 823 9
842 841 23
843 844 16
845 846 25
844 847 14
848 849 26
829 843 17
849 850 11
846 831 23
851 852 18
850 845 28
840 842 7
853 851 17
854 848 19
855 856 28
857 858 26
859 860 21
861 862 11
863 864 15
864 865 29
860 866 12
856 867 6
868 853 7
869 863 21
870 871 20
862 855 9
872 873 15
874 875 25
876 877 9
878 868 23
879 880 15
881 857 22
882 883 8
884 861 28
885 882 17
886 887 20
888 889 25
875 870 9
867 876 27
890 891 28
871 886 24
880 854 27
892 881 25
866 893 14
894 895 18
895 874 18
896 890 25
897 898 17
847 899 24
900 901 15
902 896 28
898 903 28
877 904 12
883 884 30
904 879 8
905 906 16
906 872 7
865 907 6
903 908 22
889 909 25
873 910 5
901 911 22
912 905 5
913 914 10
915 916 17
917 918 17
919 885 6
920 888 15
911 912 6
921 922 24
899 920 14
887 923 5
918 915 21
924 925 15
926 924 21
927 928 26
908 927 13
923 929 28
907 878 23
930 919 29
931 932 11
929 869 23
858 933 12
934 900 5
935 931 24
936 937 14
891 938 14
939 940 16
941 942 17
938 935 17
909 943 6
944 921 15
945 897 21
946 945 8
933 947 12
948 892 5
943 944 27
922 936 14
910 949 12
950 946 5
937 951 27
932 952 19
953 939 21
949 930 24
954 955 20
956 859 9
925 941 13
957 917 16
958 959 20
914 902 20
960 913 27
942 961 24
962 963 10
964 965 25
966 960 12
965 966 18
893 957 27
940 954 23
967 968 21
968 969 5
961 970 19
955 926 24
971 964 15
972 962 21
970 972 23
973 950 11
974 975 9
976 894 22
947 971 26
969 974 23
975 953 10
951 977 13
916 976 7
928 978 17
952 973 25
963 948 29
979 980 25
981 967 28
852 981 27
982 979 5
983 982 7
984 958 14
977 985 13
978 986 14
987 956 16
980 987 10
988 989 5
986 990 18
959 991 28
989 934 29
991 992 29
993 984 9
994 995 5
990 988 13
995 996 21
997 983 28
998 997 14
999 998 19
992 999 27
985 994 24
1 993 18
996 2 28
