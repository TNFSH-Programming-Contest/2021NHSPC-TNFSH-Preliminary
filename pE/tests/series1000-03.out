17540.0000000000000000000
